site/app/templates/forum/ShowCategories.twig

```diff
@@ -0,0 +1,120 @@
+<div class="content forum_content">
+
+    {% include "forum/ForumBar.twig" with {
+        "forum_bar_buttons_right" : forumBarData.forum_bar_buttons_right,
+        "forum_bar_buttons_left" : forumBarData.forum_bar_buttons_left,
+        "show_threads" : forumBarData.show_threads,
+        "thread_exists" : forumBarData.thread_exists
+    } %}
+
+    <h3 style="margin-bottom:10px;"> Manage Categories </h3>
+
+    <div id="manage_categories_view" align="center">
+        <div style="max-width: 750px;" align="left">
+
+        <input type="hidden" name="csrf_token" value="{{ core.getCsrfToken() }}" />
```

Comment:
@bmcutler from our conversation today, I'm assuming that this is an issue (core access in twig)? It makes more sense to pass the csrf_token as a param to twig anyway.

Reply:
Relating to #3900

Reply:
If we merge in #3900 after this one, it will get corrected!